habitat_baselines/common/utils.py

```diff
@@ -0,0 +1,148 @@
+import argparse
+import glob
+import os
+from collections import defaultdict
+from typing import Optional
+
+import numpy as np
+import torch
+import torch.nn as nn
+
+from habitat import Config, get_config, make_dataset
+from habitat_baselines.config.default import get_config as baseline_cfg
+
+
+class Flatten(nn.Module):
```

Comment:
All this classes need docs, when it should be used.

Reply:
Already on TODO list :)